src/meta/meta_ckpt.c

```diff
@@ -976,6 +983,24 @@ __wt_meta_sysinfo_set(WT_SESSION_IMPL *session)
         WT_ERR(__wt_metadata_update(session, WT_SYSTEM_OLDEST_URI, buf->data));
     }
 
+    /*
```

Comment:
There is no need of multiline comment here.

Reply:
Fixed